docs/static_site/src/_config_prod.yml

```diff
@@ -41,6 +41,7 @@ youtube_username: apachemxnet
 google_analytics: UA-96378503-1
 versions: 
   - master
```

Comment:
in near future it would be great to map master branch content and v1.x content to the /versions/master and /versions/v1.x paths respectively.